services/api-dotnet/src/app/Api.NoSql/Controllers/UsersController.cs

```diff
@@ -64,8 +66,50 @@ public async Task<IActionResult> UpdateCurrentAsync([FromBody] UpdateCurrentUser
                 user.FirstName,
                 user.LastName,
                 user.Email,
-                user.IsEmailVerified
+                user.IsEmailVerified,
+                user.AvatarUrl
+            });
+        }
+
+        [HttpPost("upload-photo")]
+        public async Task<IActionResult> UploadCurrentPhoto([FromForm] UploadPhotoModel model)
+        {
+            await using var stream = model.File.OpenReadStream();
+
+            var avatarUrl = await _cloudStorageService.UploadPublicAsync(
+                $"avatars/{CurrentUserId}-{DateTime.UtcNow:s}-{model.File.FileName}",
```

Comment:
I think, there's no need to put datetime into file name. If we need uniqueness, it'd be better to generate id (but there's no need for it). So user id + file name should be enough.

Reply:
Filename generation logic copied from node.js version